charm/docs/index.md

```diff
@@ -1,64 +1,106 @@
+* Modify Flask-specific features like a secret key for security-related needs, the run environment
+  (e.g., production) or where the application is mounted.
+
+See the [Actions](https://charmhub.io/repo-policy-compliance/actions) and
+[Configurations](https://charmhub.io/repo-policy-compliance/configurations) tabs to learn more about the
+actions and configurations supported by this charm.
+
+**Contents**
+1. [Policies](#policies)
 2. [Get started](#get-started)
-    1. [What you'll need](#what-youll-need) 
+    1. [Requirements](#requirements) 
     2. [Set up](#set-up)
     3. [Deploy](#deploy)
     4. [Generate an authentication token and test](#token)
 3. [Integrations](#integrations)
-4. [Next steps](#next-steps)
-    1. [Learn more](#learn-more)
-    2. [Join the community](#join-the-community)
-5. [License](#license)
+4. [Learn more](#learn-more)
+5. [Project and community](#project-and-community)
+6. [License](#license)
 
-------------------------------------------------------------------------------------------------
-
-# repo-policy-compliance <a name="introduction"></a>
-
-A Juju charm of a Flask application to check if a GitHub repository aligns with the policies for workflow runs. This charm consists of a Python package containing functions to check for compliance of various policies.
-
-This charm works in the context of the `github-runner` charm and should be deployed and configured for the self-hosted runners' use in OpenStack mode. The self-hosted runners execute runs in an internal environmental and can execute arbitrary code; the `repo-policy-compliance` charm verifies that the GitHub repository is set up for compliance and that only authorised code is executed. For more information, read the [GitHub runner charm documentation](https://charmhub.io/github-runner). 
 
-Like any Juju charm, this charm supports one-line deployment, configuration, integration, scaling, and more. For `repo-policy-compliance`, this includes:
-* Customising enabled policies 
-* Running in debug mode
-* Choosing different GitHub authentication methods
-* Modifying Flask-specific features like a secret key for security-related needs, the run environment (e.g., production) or where the application is mounted
-
-See the [Actions](https://charmhub.io/repo-policy-compliance/actions) and [Configurations](https://charmhub.io/repo-policy-compliance/configurations) tabs to learn more about the actions and configurations supported by this charm.
-
-## Get started <a name="get-started"></a>
-### What you'll need <a name="what-youll-need"></a>
-* A Kubernetes cloud.
-* Juju 3 installed and a controller created.
+------------------------------------------------------------------------------------------------
 
-### Set up <a name="set-up"></a>
+## Policies 
+
+The charm exposes several functions to check for compliance with the following
+policies:
+
+* `target_branch_protection`: That the branch targeted by a pull request has
```

Comment:
There is also `disallow_fork`, which can be enabled by config option, and does not allow running from external forks: https://github.com/canonical/repo-policy-compliance/blob/main/charm/charmcraft.yaml#L52 . Enabling this config option causes pull requests to be checked for this (and only this, all other things like `target_branch_protection`,... are not checked when this is enabled).